### ISSUE TITLE
Ctrl+Shift+E unexpected behavior

### ISSUE THREAD

**Issue by asherber** [microsoft/vscode#116886]
Issue Type: <b>Bug</b>

I was able to reproduce this on two different computers, with extensions disabled.

1. Press Ctrl+Shift+G: Git panel opens
1. Press Ctrl+Shift+D: Debug panel opens
1. Press Ctrl+Shift+X: Debug panel opens
1. Repeat as needed to prove these are working
1. Press Ctrl+Shift+E: Nothing happens (should open Explorer panel)
1. Press Ctrl+B: Build task input window opens (should hide side panel; Build task is Ctrl+Shift+B)
1. Some other Ctrl+ key combinations also don't work as expected at this point

It's as though Ctrl+Shift+E has somehow made the Shift key sticky. This behavior persists until I switch focus to another application and then back to VS Code; at that point, Ctrl+B correctly hides the side panel instead of opening Build task. Pressing Ctrl+Shift+E again invokes the "sticky" behavior.

VS Code version: Code 1.53.2 (622cb03f7e070a9670c94bae1a45d78d7181fbd4, 2021-02-11T11:48:04.245Z)
OS version: Windows_NT x64 10.0.19041

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-6400 CPU @ 2.70GHz (4 x 2712)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.94GB (2.30GB free)|
|Process Argv|--disable-extensions --crash-reporter-id a7044b62-6bda-4542-8c5e-7636a96cb626|
|Screen Reader|no|
|VM|0%|
</details>Extensions disabled<details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspyt653:30253241
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30256859
vstry914:30256638
pythonvsdeb440:30248342
pythonvsded773:30248341
vstes627:30244334
pythonvspyt875:30251590

```

</details>

**Comment by gjsjohnmurray** [comment]
I suggest you look at the advice at https://github.com/microsoft/vscode/wiki/Keybinding-Issues

Were the two computers completely independent? Or are you using Settings Sync between them?

**Comment by asherber** [comment]
Thanks, that was very helpful. The keyboard logging indicated (confirmed by Carnac) that Ctrl+Shift+E was being translated into MediaNextTrack. Although the two computers I tested on are completely independent, they both have Volume<sup>2</sup> installed. This turns out to be one of it's keymappings; I didn't know that because I have never used those hotkeys. VS Code behaved as expected when I removed those mappings.

Closing as user error.
